src/Vehicle/Vehicle.cc

```diff
@@ -459,6 +459,10 @@ Vehicle::~Vehicle()
     delete _mav;
     _mav = nullptr;
 
+    if (_joystickManager) {
```

Comment:
If a Vehicle is being destructed after the joystick manager is gone then that indicates some sort of worse internal coding error. All Vehicles should be gone by the time toolbox tools are destroyed. If this fixed the crash then it's just hiding a larger shutdown ordering problem.

In looking at Vehicle constructor I don't understand with the joystick manager is passed in to it. That should just be pulled from the toolbox.

Reply:
Wait, am I reading this wrong? Is this needed for the offline vehicle which has a null pointer here? Something here still seems out of whack.

Reply:
> Is this needed for the offline vehicle which has a null pointer here?

Correct

Reply:
Got it, thanks.